docker/docker-compose.yaml

```diff
@@ -0,0 +1,170 @@
+version: '3'
+services:
+    zookeeper:
+        image: wurstmeister/zookeeper
+        networks:
+            - backend
+        expose:
+            - "2181"
+        restart: unless-stopped
+    kafka:
+        image: wurstmeister/kafka:0.10.2.0
+        environment:
+            KAFKA_ADVERTISED_HOST_NAME: kafka
+            KAFKA_ADVERTISED_PORT: "9092"
+            KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock
+        networks:
+            - backend
+        expose:
+            - "9092"
+        depends_on:
+            - zookeeper
+        restart: unless-stopped
+    postgres:
+        image: postgres:9.6
+        environment:
+            POSTGRES_PASSWORD: $POSTGRES_PASSWORD
+        volumes:
+            - ./init-dendrite-dbs.sh:/docker-entrypoint-initdb.d/init-dendrite-dbs.sh
+            - $HOME/dendrite/postgres:/var/lib/postgresql/data
+        networks:
+            - backend
+        ports:
+            - "5432:5432"
```

Comment:
The only reason I publish the port on the host here (which is what the 'ports' directive does) is to be able to use `create-account`. You can change it, but you'll have to adjust the database URIs in `dendrite-config.yaml` appropriately.